packages/astro-rss/test/pagesGlobToRssItems.test.js

```diff
@@ -75,11 +75,45 @@ describe('pagesGlobToRssItems', () => {
 						frontmatter: {
 							title: undefined,
 							pubDate: phpFeedItem.pubDate,
-							description: phpFeedItem.description,
+							description: undefined,
 						},
 					})
 				),
 		};
 		return chai.expect(pagesGlobToRssItems(globResult)).to.be.rejected;
 	});
+
+	it('should not fail on missing "title" key if "description" is present', () => {
+		const globResult = {
+			'./posts/php.md': () =>
+				new Promise((resolve) =>
+					resolve({
+						url: phpFeedItem.link,
+						frontmatter: {
+							title: undefined,
+							pubDate: phpFeedItem.pubDate,
+							description: phpFeedItem.description,
+						},
+					})
+				),
+		};
+		return chai.expect(pagesGlobToRssItems(globResult)).to.not.be.rejected;
+	});
+
+	it('should fail on missing "description" key if "title" is present', () => {
```

Comment:
Minor nit: I *think* `should fail` should be `should not fail` to correctly match the behavior of the test.

Reply:
Yes you're right! Feel free to submit a PR

Reply:
Done. See #9646